package.json

```diff
@@ -19,11 +19,12 @@
     "clean": "rimraf _site tests/_artifacts",
     "format": "prettier --write .",
     "lint": "prettier --check .",
-    "postinstall": "run-s setup:husky setup:pdfs",
+    "postinstall": "run-s setup:husky setup:pdfs setup:si-svgo-config",
     "serve": "anywhere -p 8080 -d ./_site",
     "serve:watch": "run-p build:watch serve",
     "setup:husky": "is-ci || husky install",
     "setup:pdfs": "npm install https://github.com/simple-icons/simple-icons-pdf/tarball/master --no-save",
+    "setup:si-svgo-config": "sdown https://raw.githubusercontent.com/simple-icons/simple-icons/develop/svgo.config.js node_modules/simple-icons/svgo.config.js",
```

Comment:
Since we already have the project installed can't we just import the config file from `node_modules` and re-export it in `svgo.config.js`?

Reply:
The `svgo.config.js` file is not published to npm, and it doesn't make sense to do so just so we can use it here.

That said, I'd prefer to just add a `svgo.config.js` file here. I see no reason why the SVGO config of this and the main repo _must_ be the same. I see no upside to pulling it in like this (and only potential downside if e.g. if it doesn't work for someone).

Reply:
Maintain another synchronization of files? Well, don't count on me for that 🙌🏼